src/shared/components/palette/dynamic-multi-select-list/write-dynamic-multi-select-list-field.html

```diff
@@ -0,0 +1,23 @@
+<div class="form-group bottom-30" [ngClass]="{'error': !checkboxes.valid && checkboxes.touched}" [id]="id()">
+
+    <fieldset>
+  
+      <legend>
+        <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
+        <span *ngIf="caseField.hint_text" class="form-hint">{{caseField.hint_text}}</span>
+        <span *ngIf="checkboxes.errors && checkboxes.touched" class="error-message">{{checkboxes.errors | ccdFirstError}}</span>
+      </legend>
+  
+      <ng-container *ngFor="let checkbox of caseField.list_items">
+  
+        <div class="multiple-choice">
+          <input class="form-control" id="{{ buildElementId(checkbox.code) }}" [name]="id()" type="checkbox" [value]="checkbox.code" [checked]="isSelected(checkbox.code)" (change)="onCheckChange($event)">
```

Comment:
Can `[name]="id()"` be replaced with `name="{{ id() }}"` like you've done for the `id` attribute? :)